examples/ui-demo/contracts/src/ERC20Mintable6Decimals.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.26;
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+contract ERC20Mintable6Decimals is ERC20 {
+    uint8 private immutable _decimals;
+
+    constructor(string memory name, string memory symbol)
+        ERC20(name, symbol)
+    {
+        _decimals = 6;
```

Comment:
Added this because I need an erc20 token that has 6 decimals so that our policy can accurately price the token with the `priceReferenceTokenAddress` set to usdc on mainnet

Reply:
Also, I didn't just edit the other erc20 contract we have to accept a decimal parameter because I didn't want to mess up the deployment address of that contract.  Not sure if that matters but did that just in case